paddle/fluid/pybind/op_function_generator.cc

```diff
@@ -23,45 +23,47 @@
 #include "paddle/fluid/pybind/pybind.h"
 #include "paddle/fluid/string/string_helper.h"
 
+// clang-format off
 const char* OUT_INITIALIZER_TEMPLATE =
     R"({"%s", {std::shared_ptr<imperative::VarBase>(new imperative::VarBase(tracer->GenerateUniqueName()))}})";
 
 const char* OP_FUNCTION_TEMPLATE =
-    R"([](const imperative::NameVarBaseMap& ins, const framework::AttributeMap& attrs, 
-    imperative::NameVarBaseMap outs, const std::map<std::string, size_t>& out_nums)
-  {
-    auto tracer = imperative::GetCurrentTracer();
-    if (outs.size() == 0) {
-      if (out_nums.size() == 0) {
-        imperative::NameVarBaseMap outs_ = %s;
-        outs = std::move(outs_);
-      } else {
-        for (auto &pair : out_nums) {
-          for (size_t i = 0; i < pair.second; i ++) {
-            auto var_base_name = tracer->GenerateUniqueName();
-            auto out = new imperative::VarBase(var_base_name);
-            outs[pair.first].emplace_back(std::shared_ptr<imperative::VarBase>(out));
-          }
+R"(
+imperative::NameVarBaseMap %s(const imperative::NameVarBaseMap& ins, const framework::AttributeMap& attrs, 
```

Comment:
Add `inline` to avoid function redefinition when this header file is included directly or indirectly twice.

Reply:
Done, thanks.